kong.conf.default

```diff
@@ -1826,3 +1826,29 @@ G# -----------------------
 
 #wasm_filters_path =              # Path to the directory containing Wasm filters
                                   # that Kong must load on startup.
+
+#wasm_shm_kv =                    # Define a shared key/value memory zone accessible
+                                  # through all Contexts and by all nginx worker
+                                  # processes.
+
+#wasm_shm_queue =                 # Define a shared queue memory zone accessible
+                                  # through all nginx contexts and by all nginx
+                                  # worker processes.
+
+#wasm_tls_no_verify_warn =        # Toggle the TLS verification warning log for
+                                  # Wasm sockets. 
+
+#wasm_socket_buffer_reuse =       # Toggle Wasm sockets buffer reuse.
+
+#wasm_socket_buffer_size =        # Set a buffer `size` for reading response payloads
+                                  # with Wasm sockets.
+
+#proxy_wasm_request_headers_in_access =   # Toggles the "on_request_headers in
+                                          # access phase" WasmX feature. When
+                                          # enabled, this flag will cause the
+                                          # `on_request_headers` phase to be
+                                          # executed in the `access` phase instead
+                                          # of the `rewrite` phase of Nginx.
```

Comment:
This seems like a product decision that we should make rather than asking the user to decide on such behavior.
We can't expect our users to understand such Nginx internal details.

What do you think?

Reply:
This is not intended for exposure to the Gateway users, imho should be removed.